### DISCUSSION TITLE
DNS-O-Matic Malformed Password

### DISCUSSION THREAD

**Discussion by muslimsteel** [qdm12/ddns-updater#406] [Q&A] [answered]
Hello, I am trying to get DNSOMatic to update using this updater, but for whatever reason, it is giving me the error malformed password. I tried turning on debug logging but all that did was add the config file to the logs and then it shuts down. Am I missing something? My password is 20 characters long and only has one special character in it a "$" sign, wondering if that was an issue.
`{
      "provider": "dnsomatic",
      "host": "*",
      "username": "***@gmail.com",
      "password": "***",
      "ip_version": "ipv4"
    }`

Thanks for the help!

**Comment by qdm12** [comment, marked as answer]
This was fixed with commit https://github.com/qdm12/ddns-updater/commit/f8bb927de621e654f2637d0c20eb36db823dbd04
